src/hooks/mention.test.ts

```diff
@@ -18,7 +18,7 @@ describe('Chat Mention Hooks', () => {
   });
 
   describe('useMentions', () => {
-    const currentMemberRoute = `/${buildGetCurrentMemberRoute()}`;
+    const currentMemberRoute = buildGetCurrentMemberRoute();
```

Comment:
Not against, but maybe we want to do this together with all the other routes. Should we do it in another PR?